src/managers/translation_manager.ts

```diff
@@ -19,11 +20,11 @@ export class TranslationManager {
     public get_translation(scope: string, lang: string, entity: string, key: string): string {
         return this.data[scope][lang][entity + "__" + key] || "N/A";
     }
-    public get_sms(key: string): string {
-        return this.data["sms"]["he"]["nx_devices__" + key] || "N/A";
+    public get_sms(key: string, options?: TranslationOptions): string {
```

Comment:
I think it's unnecessary to introduce here TranslationOptions type.
better to keep function signature as (lang, entity, key). but idea used in TranslationOptions was good. So we can use it for EntityOptions and LangOptions types